package.json

```diff
@@ -28,5 +28,7 @@
     "sass-loader": "^7.1.0",
     "vue-template-compiler": "^2.6.7"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@glidejs/glide": "^3.2.6"
```

Comment:
Why has this been installed? It doesn't seem to be used anywhere?

Reply:
This actually needs to be taken out. 
Installed for carousel but that with any JS can be like a phase two. 
Will remove now and re-request the merge.